### DISCUSSION TITLE
Deploy Client with default settings changed

### DISCUSSION THREAD

**Discussion by st0rm-cr0w** [rustdesk/rustdesk-server#323] [Q&A] [answered]
Hello,

We're looking to implement RustDesk in our company and **have set up a working PoC** (with the OSS version) without too much of an issue.
However, we're concerned about RustDesk being used to access other computers instead of being used to give support from our IT Department.

We were looking to purchase the Pro version after the good experience we've had with the tool, however, we were wondering if the client can be configured to change the default settings. 
I've read the documentation, and while the Pro version's webpanel mentions customizing clients, I believe it only affects the network configuration.

We'd like to use the tool for giving support to our employees, and were looking to change the default **Security** settings the Client comes bundled with, **namely, whitelisting only certain addresses** so the employee's computers can only be accessed by our IT Department and not by other RustDesk clients.

Is it something that can be done with the Pro version? If not, is it something that's planned or can be done in some other way (instead of individually going to each computer and setting it up)?

Kind regards

**Comment by rustdesk** [comment, marked as answer]
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/strategy/
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/permissions/
https://rustdesk.com/docs/en/self-host/rustdesk-server-pro/console/#assign-device-users-and-groups-to-devices

**Reply by st0rm-cr0w** [reply]
Ah, it seems I was looking at the wrong part of the documentation, thank you for clearing it up. 
Sorry!
